### ISSUE TITLE
Removed "Settings saved .. " text in epub downloaded from mtlnovel.com

### ISSUE THREAD

**Issue by seizue** [dteviot/WebToEpub#727]
Removed "Settings saved .." text in epub downloaded from mtlnovel.com

These text always show up from epub novels downloaded from mtlnovel.com when using Moon + Reader (mobile app). I notice the text is not visible when using Lithium app but when using Moon + Reader, it always show up in the beginning of every chapter. This issue only apper from epub files downloaded from mtlnovel.com, other sites are are fine.

![Screenshot_2022-04-28-11-54-27-38](https://user-images.githubusercontent.com/25120376/165678153-f370aa16-637f-47bd-b963-f6913285f8d0.jpg)

**Comment by dteviot** [comment]
@seizue 
Seems to be an element marked hidden on each chapter.  I guess Moon + Reader does not recognize the hidden attribute.
You might like to tell them about it.

That said, I've updated WebToEpub to remove these elements.

Test versions for Firefox and Chrome have been uploaded to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing. Pick the one suitable for you, follow the "How to install from Source (for people who are not developers)" instructions at https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode#user-content-how-to-install-from-source-for-people-who-are-not-developers and let me know how it goes.

Tested against a couple of chapters from https://www.mtlnovel.com/my-female-ghost-is-super-fierce/

For my notes: Time taken.  15 minutes.
